lib/plugin.js

```diff
@@ -115,7 +115,13 @@ internals.schema.strategy = Joi.object({
         timeSkewSec: Joi.number().integer().min(0).default(0)
     })
         .when('.validate', { is: Joi.not(false), then: Joi.allow(false) })
-        .required()
+        .required(),
+
+    headerName: Joi.alternatives()
```

Comment:
The other entries seems to have been defined in alphabetical order, so it would be nice to continue that.

Reply:
oh I didn't notice that. Will do that